ports/librdkafka/vcpkg.json

```
@@ -1,9 +1,18 @@
 {
   "name": "librdkafka",
-  "version-string": "1.7.0",
-  "port-version": 2,
+  "version-string": "1.8.2",
```

```suggestion
  "version": "1.8.2",
```
This might be served by [version](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version).